client/jetpack-connect/style.scss

```diff
@@ -4,6 +4,13 @@
 			max-width: 600px;
 		}
 	}
+	
+	.wpcom-colophon { 
```

Comment:
Could we place this somewhere else, under a different classname? I'd like to keep the styles in `is-section-*` as minimal as possible (ideally, I'd love to see them not depend on that classname at all).

Reply:
:+1: